internal/sys/fs.go

```diff
@@ -301,25 +301,16 @@ func stdioWriter(w io.Writer, defaultStat stdioFileInfo) *FileEntry {
 }
 
 func stdioStat(f interface{}, defaultStat stdioFileInfo) fs.FileInfo {
-	if f, ok := f.(*os.File); ok && platform.IsTerminal(f.Fd()) {
-		return stdioFileInfo{defaultStat[0], modeCharDevice}
+	if f, ok := f.(*os.File); ok {
+		mode := modeCharDevice
```

Comment:
so as far as I understand, this will fallback to a char device if there's an error getting stat. I would prefer to change the signature to propagate the error. It might take changing multiple functions, but I think it doesn't have end user API impact.

Reply:
ok will try